src/System.Security.Cryptography.Xml/tests/SignedXmlTest.cs

```diff
@@ -284,7 +284,7 @@ public void AsymmetricRSAMixedCaseAttributesVerifyWindows()
             Assert.True(v1.CheckSignature());
         }
 
-        [Fact(Skip = "https://github.com/dotnet/corefx/issues/16691")]
+        [Fact]
         public void AsymmetricDSASignature()
```

Comment:
Well, the DSA type is there, and it works everywhere else.  When the macOS version gets to a part where it needs to call SecKeyGeneratePair it throws a `PlatformNotSupportedException`, yeah.  I'm just making sure we avoid a conflict due to the PNSE :smile:.